sdk/Sdk.Generators/Constants.cs

```diff
@@ -14,6 +14,8 @@ internal static class BuildProperties
         {
             internal const string EnableSourceGen = "build_property.FunctionsEnableMetadataSourceGen";
             internal const string EnablePlaceholder = "build_property.FunctionsEnableExecutorSourceGen";
+            internal const string AutoRegisterGeneratedFunctionsExecutor = "build_property.FunctionsAutoRegisterGeneratedFunctionsExecutor";
```

Comment:
Should we use the `FunctionsEnable` prefix to be consistent with the other 2 properties? Any suggestions for the names?

Reply:
The name makes sense to me as is and I don't think enable would add much clarity - it would also make the property name very long.

Reply:
Maybe `FunctionsEnableGeneratedExecutorRegistration`